airflow/.env

```diff
@@ -0,0 +1,7 @@
+ASK_ASTRO_ENV='local'
```

Comment:
I feel it bit risky since in the local dev it may contain actual credentials and someone would accidentally push it to GitHub

Reply:
How is this change resolved?